backend/tests/load_testing/scenarios/chat_load_test.py

```
@@ -0,0 +1,118 @@
+    if stats.total.num_requests > 0:
+        failure_rate = (stats.total.num_failures / stats.total.num_requests) * 100
+        logger.info(f"  Failure Rate: {failure_rate:.2f}%")
+        logger.info(f"  Avg Response Time: {stats.total.avg_response_time:.0f}ms")
+        logger.info(f"  P95 Response Time: {stats.total.get_response_time_percentile(0.95):.0f}ms")
+        logger.info(f"  P99 Response Time: {stats.total.get_response_time_percentile(0.99):.0f}ms")
+    
+    logger.info("=" * 60)
+
+
+@events.request.add_listener
+def on_request(request_type, name, response_time, response_length, response, context, exception, **kwargs):
```

Ruff (0.14.10)

Unused function argument: `response_time`

(ARG001)